File: Exercise/Medium/03. Factorial Calculation/Code/Factorial Calculation.cs
using System;

class Program {
    static void FactorialCalculation(int num) {
        int factorial = num;
        for (int i = num - 1; i > 0; i--) {
            factorial *= i;
        }
        Console.WriteLine(factorial);
    }

    static void Main() {
        int num = int.Parse(Console.ReadLine());
        FactorialCalculation(num);
    }
}

File: Exercise/Medium/03. Factorial Calculation/Code/Factorial Calculation.ts
function FactorialCalculation(num :number) :void {
    let factorial :number = num;
    for (let i :number = num - 1; i > 0; i--) {
        factorial *= i;
    }
    console.log(factorial);
}

FactorialCalculation(5);
FactorialCalculation(3);